projects/colorful-dots/index.md

```diff
@@ -4,18 +4,18 @@ title: Colorful Dots
 authors: [Julia Hilt]
 authors-url: [https://github.com/piixelcat]
 ---
-# Colorful Dots
+## Colorful Dots
```

Comment:
ähm. I didn't mean to move it one h lower. The Title will be added by the layout. That means there will be a H1 for colorful dots. So you could remove this all together and keep the levels of headlines as they where.